src/class-wpml-gutenberg-integration.php

```diff
@@ -201,20 +215,29 @@ private function render_block( $block ) {
 	 * It should really return the HTML before and after the blocks
 	 * We're just guessing what it is here
 	 * The usual innerHTML would be: <div class="xxx"></div>
-	 * So we try to split at ></
+	 * The columns block also includes new lines: <div class="xxx">\n\n</div>
+	 * So we try to split at ></ and also include white space and new lines between the tags
 	 *
-	 * @param $inner_HTML
+	 * @param array $block
 	 *
 	 * @return array
 	 */
-	private function guess_inner_HTML_parts( $inner_HTML ) {
-		$parts = explode( '></', $inner_HTML );
-		if ( count( $parts ) === 2 ) {
-			$parts[0] .= '>';
-			$parts[1] = '</' . $parts[1];
-		} else {
-			// Can't guess so return innerHTML as first part and empty for second part
-			$parts = array( $inner_HTML, '' );
+	private function guess_inner_HTML_parts( $block ) {
+		$inner_HTML = $block['innerHTML'];
+
+		$parts = array( $inner_HTML, '' );
+
+		preg_match( '#>\s*</#', $inner_HTML, $matches );
+
+		if ( count( $matches ) === 1 ) {
+			$parts = explode( $matches[0], $inner_HTML );
+			if ( count( $parts ) === 2 ) {
+				$match_mid_point = 1 + ( strlen( $matches[0] ) - 3 ) / 2;
```

Comment:
@brucepearson, I suggest using `mb_strlen` and `mb_substr`. It might not be a problem now, but maybe in the future.